### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
N/A

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs

* **What is the current behavior?** (You can also link to an open issue here)
Docs weren't updated.

* **What is the new behavior (if this is a feature change)?**
Updated README to reflect the changes that are implemented.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
None

* **Other information**:

### CODE REVIEW

File: gitcache/README.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
-1. Active - Checks for the last commit in 90 days. This is can be fetched from the above blob storage.
+Scorecard checks for these don't need GitHub API. It requires a Git API.
+1. Active - Checks for the last commit in 90 days. This can be fetched from the above blob storage.
 1. Frozen-Deps - Checks for a file with in the tarball.
 1. CodeQLInCheckDefinitions - Checks for a file content within the tarball.
 1. Security-Policy - Checks for a file name within the tarball.
 1. Packaging - Checks for a file content within the tarball.
 
-The number of checks within scorecard https://github.com/ossf/scorecard#checks is `14` out of which `12` use GitHub API's. The two that does not use the GitHub API's are `Fuzzing` and `CII`.
+The number of checks within scorecard https://github.com/ossf/scorecard#checks is `14` out of which12` use GitHub API's. The two that do not use the GitHub APIs are `Fuzzing` and `CII`.
```
Comment (inferno-chromium) [inline gitcache/README.md:35]: space between which and 12

File: gitcache/README.md [hunk 2]
```diff
@@ -1,62 +1,62 @@
 # gitcache - Reducing the GitHub API usage for scaling scorecard
 
 
-To scale 1000's of repositories it is important that the codebase is cognizant on the usage of the GitHub API's.
+To scale 1000's repositories, the codebase must be aware of the GitHub API's usage.
 
-One of the stratergies to avoid using the API directly cloning the repository.
+One of the strategies to avoid using the API directly cloning the repository.
```
Comment (inferno-chromium) [inline gitcache/README.md:6]: directly is cloning

**Review by inferno-chromium — APPROVED** [review]

File: gitcache/README.md (hunk 2 above)
Comment (naveensrinivasan) [reply gitcache/README.md]: Thanks! Go it

File: gitcache/README.md (hunk 1 above)
Comment (naveensrinivasan) [reply gitcache/README.md]: Got it! Thanks

**Pull request merged**